src/fixate/drivers/handlers.py

```diff
@@ -0,0 +1,50 @@
+"""
+This module implements concrete AddressHandler type, that
+can be used to implement IO for the fixate.core.switching module.
+"""
+from __future__ import annotations
+
+from typing import Sequence
+
+from fixate.core.switching import Pin, PinValueAddressHandler
+from fixate.drivers import ftdi
+
+
+class FTDIAddressHandler(PinValueAddressHandler):
+    """
+    An address handler which uses the ftdi driver to control pins.
+
+    We create this concrete address handler because we use it most
+    often. FT232 is used to bit-bang to shift register that are control
+    the switching in a jig.
+    """
+
+    def __init__(
+        self,
+        ftdi_description: str,
+        pins: Sequence[Pin] = tuple(),
+    ) -> None:
+
+        self.pin_list = tuple(pins)
+        # call the base class super _after_ we create the pin list
```

Comment:
This comment is probably not really relevant now that the pin generation stuff isn't here now.

Reply:
I've modified the comment a little, but it is still relevant that pin_list is assigned before calling super.

But now I'm thinking I should also change the base class so you always pass the pin list into the constructor, instead of the "subclass + declarative pin list" that we've had in the past.